src/main/java/nl/thehyve/ocdu/models/UploadSession.java

```diff
@@ -6,6 +6,7 @@
 /**
  * Created by piotrzakrzewski on 28/04/16.
  */
+
```

Comment:
You can avoid committing such unnecessary whitespace changes by either checking out  version of the file from the master / previous commit with git checkout COMMIT /path/to/file
or more simply by doing ctrl+z in intelliJ. You always see green bars on lines which were modified with respect to HEAD.